packages/vue/src/cypress/FFileItem.pageobject.ts

```diff
@@ -7,36 +7,70 @@ import { FProgressbarPageObject } from "./FProgressbar.pageobject";
 export class FFileItemPageObject implements BasePageObject {
     public selector: string;
     public el: () => DefaultCypressChainable;
-    public fileName: () => DefaultCypressChainable;
-    public cancelDeleteButton: () => DefaultCypressChainable;
-    public cancelDeleteButtonIcon: () => DefaultCypressChainable;
-    public progressMeter: FProgressbarPageObject;
-    public fileIcon: () => DefaultCypressChainable;
+
+    /**
+     * @internal
+     */
+    public _progressMeter: FProgressbarPageObject;
 
     /**
      * @param selector - the root of the file item, usually `<div class="file-item">...</div>`.
      */
     public constructor(selector: string) {
         this.selector = selector;
         this.el = () => cy.get(this.selector);
-        this.fileName = () => cy.get(`${this.selector} .file-item__file-name`);
-        this.fileIcon = () =>
-            cy.get(`${this.selector} .file-item__file-open .icon`);
-        this.cancelDeleteButton = () =>
-            cy.get(`${this.selector} .file-item__file-remove`);
-        this.cancelDeleteButtonIcon = () =>
-            cy.get(`${this.selector} .file-item__file-remove .icon`);
-        this.progressMeter = new FProgressbarPageObject(this.selector);
+        this._progressMeter = new FProgressbarPageObject(this.selector);
     }
 
-    public typeOfFileIcon(): Cypress.Chainable<string> {
-        return this.fileIcon().then((el) =>
+    /**
+     * @internal
+     */
+    public _cancelDeleteButton(): Cypress.Chainable<JQuery<HTMLElement>> {
+        return cy.get(`${this.selector} .file-item__file-remove`);
+    }
+
+    /**
+     * @internal
+     */
+    public _cancelDeleteButtonIcon(): Cypress.Chainable<JQuery<HTMLElement>> {
```

Comment:
Delete-knappen slottas in av våra exempel och jag tycker inte det är rimligt att man försöker testa ikonen. Dessutom tror jag detta är trasigt (vi använder det inte själva ens), den kommer alltid matcha fel ikon (new window) istället för den man nog avser matcha.